src/alto-ui/Datagrid/components/DatagridCellInput/DatagridCellInput.js

```diff
@@ -10,147 +17,91 @@ import './DatagridCellInput.scss';
 const ENTER_KEY_CODE = 13;
 const ESC_KEY_CODE = 27;
 
-class DatagridCellInput extends React.Component {
-  constructor(props) {
-    super(props);
+const DatagridCellInput = props => {
```

Comment:
```suggestion
const DatagridCellInput = ({ editing,
    modifiers,
    onChange,
    onStopEditing,
    onStartEditing,
    value,
    type}) => {
```

Reply:
Sorry, but then tests are failed.
In some cases we need eg. `onChange` and `handleChange`. 
After destructurization we loose them.
We need each prop for `getInputProps`